zmon_aws_agent/aws.py

```diff
@@ -392,6 +397,7 @@ def get_running_elbs_classic(region, acc):
     return lbs
 
 
+@trace(operation_name='get_running_elbs_application', tags={'aws', 'elb'})
```

Comment:
operation_name not needed here + tags should be a dict

Reply:
oops, missed that one :/